lib/exogenesis/passengers/vim_plug.rb

```diff
@@ -0,0 +1,18 @@
+require 'exogenesis/support/passenger'
+
+class VimPlug < Passenger
+  register_as :vim_plug
+  with_emoji :gift
+
+  def up
+    execute_interactive 'Installing and Updating Vim Bundles', 'vim +PlugInstall\! +qall'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.